Sources/Bow/Data/Try.swift

```diff
@@ -22,22 +22,22 @@ public typealias TryOf<A> = Kind<ForTry, A>
 
 /// Describes the result of an operation that may have thrown errors or succeeded. The type parameter corresponds to the result type of the operation.
 public final class Try<A>: TryOf<A> {
-    private let value: _Try<A>
+    fileprivate let value: Either<Error, A>
```

Comment:
Like in the case of `Validated`, I prefer maintaining its internal representation.